### DISCUSSION TITLE
Having a br tag in a button doesn't really apply for i tag content

### DISCUSSION THREAD

**Discussion by MordechaiHadad** [saadeghi/daisyui#1086] [Q&A] [answered]
let's say we have this code:
```html
	<button class="btn">
        Test
		<br />
		<i class="fa-solid fa-arrow-down" />
	</button>
```
What you get is 
![img](https://p0rn.pro/ADeepAndThoughtfulProcess)
What is the solution to fix this?

**Comment by saadeghi** [comment, marked as answer]
`btn` has `flex` style to put its content next to each other so if you want to show the content vertically there's no need for a `<br>`. You can just add `flex-col` class to `btn`  

https://play.tailwindcss.com/Dc9o3zyzUP
```html
<button class="btn flex-col gap-1">
  Test
  <i class="fa fa-arrow-down"></i>
</button>
```
